jck/runtime.vm/playlist.xml

```diff
@@ -68,7 +48,7 @@
 	-test=Jck -test-args=$(Q)tests=vm/jdwp,jckRoot=$(JCK_ROOT),jckversion=$(JCK_VERSION),testsuite=RUNTIME$(Q); \
 	$(TEST_STATUS)</command>
 		<levels>
-			<level>sanity</level>
+			 /level>
```

Comment:
This line is incompleted. Please fix it.

Reply:
Fixed! Thankns for the catch.